docs/Course Wiki/PHYS Course Offerings/PHYS212.md

```diff
@@ -0,0 +1,59 @@
+# PHYS212 
+
+PHYS212(University Physics: Electricity and Magnetism) is a 4-credit hour course that is required for ECE majors as a part of the Foundational Mathematics and Science requirement. It is also required for most other engineering majors, including Computer Science majors. It is offered in the fall, spring, and summer semesters.
+
+## Content Covered
+
+- Coulomb’s Law, Electrostatics, Electric Field
+- Electric Flux, Gauss’s Law, Electric Potential
+- Capacitors and Capacitance
+- Current and DC Circuit Analysis, KVL, KCL, Ohm’s Law
+- RC Circuits
+- Magnetic Force and Field, Biot-Savart and Ampere’s Law
+- Magnetic Induction, Faraday’s Law, Motional EMF
+- Inductors and RL Circuits
+- LC and LRC Circuits, Harmonic Oscillation, Phasors
+- AC Circuits, Resonance, Power, Transformers
+- Maxwell’s Equations, Electromagnetic Waves
+- Polarization, Intensity, Geometric Optics
+
+PHYS212 is a solid foundational course in many concepts absolutely essential for any ECE major. Concepts such as DC circuit analysis, phasors, resonance and power, and basic electromagnetism provide an excellent conceptual base for ECE majors to understand topics in courses such as ECE 110, 210, 330, and more.
```

Comment:
Mention ECE329 here.  Also format every class as ECE123 with no space between the subject and course number, this is just for consistent formatting.